### PR TITLE
feat(images): document image release import

### PR DESCRIPTION
**Pull request by aguadoenzo**
Importing a specific image release was introduced a few months ago but never
documented

### CODE REVIEW

File: howto/manage/images.md [hunk 1]
```diff
@@ -58,6 +58,23 @@ The following command removes version `1` of the image with the name `default`:
...
+With every new Anbox Cloud release, updated images are published. By default, the latest image release is pulled by AMS, but you can request a
+specific release with the following syntax:
+
+    amc image add <local image name> <remote image name>@<release>
+
+For instance, to fetch the arm64 Android 11 image of the 1.11.2 release:
+
+    amc image foobar bionic:android11:arm64@1.11.2
+
+You can now use the `foobar` image as you would with any other image.
```
Comment (ru-fu) [inline howto/manage/images.md:72]:
```suggestion
You can then use the `foobar` image in the same way as any other image.
```

File: howto/manage/images.md [hunk 2]
```diff
@@ -58,6 +58,23 @@ The following command removes version `1` of the image with the name `default`:
...
+
+[note type="information" status="Tip"]
+Image updates contain important security patches and optimizations. Use older images only when strictly necessary.
```
Comment (ru-fu) [inline howto/manage/images.md:75]:
```suggestion
Image updates contain important security patches and optimisations. Use older images only when strictly necessary.
```

**Review by ru-fu — COMMENTED** [review]
Thanks! Just two small changes. :)

File: howto/manage/images.md [hunk 3]
```diff
@@ -58,6 +58,23 @@ The following command removes version `1` of the image with the name `default`:
...
+With every new Anbox Cloud release, updated images are published. By default, the latest image release is pulled by AMS, but you can request a
+specific release with the following syntax:
+
+    amc image add <local image name> <remote image name>@<release>
+
+For instance, to fetch the arm64 Android 11 image of the 1.11.2 release:
+
+    amc image foobar bionic:android11:arm64@1.11.2
+
+You can then use the `foobar` image as you would with any other image.
```
Comment (ru-fu) [inline howto/manage/images.md:72]:
```suggestion
You can then use the `foobar` image as you would any other image.
```
or
```suggestion
You can then use the `foobar` image in the same way as any other image.
```
"Use with" doesn't work. ;)

**Review by ru-fu — APPROVED** [review]

**Comment by ru-fu** [comment]
Shall I publish or do you want more reviews?

**Comment by aguadoenzo** [comment]
We can publish it now if that's fine with you

**Pull request merged**